src/addons/addon.cpp

```diff
@@ -388,6 +388,13 @@ void Addon::retranslate() {
 }
 
 void Addon::maybeCreateConditionWatchers(const QJsonObject& conditions) {
+  if (m_conditionsParsed) {
```

Comment:
I added this just because it with `m_conditionsParsed` there is an assumption that `maybeCreateConditionWatchers` will only be called once. That assumption was not enforced by code though so I added the enforcement... Q_ASSERT should work as well.